### PR TITLE
add markdownList array to save files into

### PR DESCRIPTION
**Pull request by caripizza**
This branch started off as being our container form branch, but then we realized we needed to add the new markdownList array to our Redux store; so the branch does that instead you're welcome.
Summary:
- add markdownList array to save files into
--> add reducer, action, selector

### CODE REVIEW

File: src/actions/markdownActions.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
 export const UPDATE_MARKDOWN = 'UPDATE_MARKDOWN';
-export const updateMarkdown = (text) => ({
+export const updateMarkdown = text => ({
   type: UPDATE_MARKDOWN,
   payload: text
 });
+
+export const CREATE_MARKDOWN = 'CREATE_MARKDOWN';
```
Comment (mehtaphysical) [inline src/actions/markdownActions.js]: this should go in a `markdownListActions.js` file

File: src/reducers/markdownListReducer.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { CREATE_MARKDOWN } from '../actions/markdownActions';
```
Comment (mehtaphysical) [inline src/reducers/markdownListReducer.js]: This file needs a test

File: src/actions/markdownActions.js (hunk 1 above)
Comment (mehtaphysical) [reply src/actions/markdownActions.js]: Also this needs a test

File: src/selectors/markdownListSelectors.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const getMarkdownList = state => {
```
Comment (mehtaphysical) [inline src/selectors/markdownListSelectors.js:1]: This file  needs a test

File: src/reducers/index.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+import markdownListReducer from './markdownListReducer';
+import markdownReducer from './markdownReducer';
+
+export default combineReducers({
+  markdownListReducer,
```
Comment (mehtaphysical) [inline src/reducers/index.js:6]: i hate this so much, but it works so :+1:

File: src/selectors/markdownSelectors.test.js [hunk 5]
```diff
@@ -3,15 +3,19 @@ import { getMarkdown } from './markdownSelectors';
 describe('markdown selectors tests', () => {
   it('gets markdown text from state', () => {
     const state = {
-      markdown: 'some string'
+      markdownReducer: {
```
Comment (MariahAdams) [inline src/selectors/markdownSelectors.test.js:6]: A reducer should be a pure function, not something that's referenced in state.

Comment (caripizza) [reply src/selectors/markdownSelectors.test.js:6]: 🤠

Comment (caripizza) [reply src/selectors/markdownSelectors.test.js:6]: c'mon we're just learning this stuff!! (trying to be as explicit as possible without confusing ourselves with naming conventions like markdown.markdown)

**Review by MariahAdams — APPROVED** [review]
Lookin' good!

**Review by AaronD87 — APPROVED** [review]
Testing looks good Cari!

**Pull request merged**